packages/navie/src/interaction-history.ts

```diff
@@ -161,35 +165,38 @@ export class HelpLookupEvent extends InteractionEvent {
 }
 
 export class ContextItemEvent extends InteractionEvent {
-  constructor(public contextItem: ContextItem, public file?: string) {
+  constructor(public promptType: PromptType, public content: string, public location?: string) {
```

Comment:
Context items will be emitted as prompt entries, so require each one here to match a known PromptType.